src/pages/auth/ui/SignUp.tsx

```diff
@@ -1,3 +1,283 @@
+  const [isCheckedList, setIsCheckedList] = useState<CheckList>({
+    termOfService: false,
+    privateInfo: false,
+  });
+  const { showModal } = useModal();
+  const navigate = useNavigate();
+
+  const handleCheckboxChange = (checkItem: string) => {
+    if (checkItem === 'termOfService') {
+      setIsCheckedList({ ...isCheckedList, termOfService: !isCheckedList.termOfService });
+    } else if (checkItem === 'privateInfo') {
+      setIsCheckedList({ ...isCheckedList, privateInfo: !isCheckedList.privateInfo });
+    } else if (checkItem === 'checkAll') {
+      if (isCheckedList.termOfService && isCheckedList.privateInfo) {
+        setIsCheckedList({
+          termOfService: false,
+          privateInfo: false,
+        });
+      } else {
+        setIsCheckedList({
+          termOfService: true,
+          privateInfo: true,
+        });
+      }
+    }
+  };
+
+  //! 회원가입 버튼 클릭
+  const handleSignupClick = () => {
+    const url = new URL(window.location.href);
+    const email = url.searchParams.get('email');
+    const nickname = url.searchParams.get('nickname');
+
+    if (
+      email &&
+      nickname &&
+      selectedJobPosition &&
+      selectedCareerYear &&
+      selectedTechStackList?.length &&
+      selectedCategoryList?.length
+    ) {
+      if (!areAllTrue(isCheckedList)) {
+        showModal({
+          content: `모든 이용약관에 동의해주세요.`,
+          type: 'alert',
+        });
+        return;
+      }
+      const userData = {
+        email,
+        nickname,
+        selectedJobPosition,
+        selectedCareerYear,
+        selectedTechStackList,
+        selectedCategoryList,
+      };
+      setUser(userData);
+
+      const userDataForBE = {
+        email,
+        nickname,
+        position_category_id: parseInt(selectedJobPosition.value),
+        carrer_category_id: parseInt(selectedCareerYear.value),
+        stack_category_id: selectedTechStackList.map(({ id }) => id),
+        project_category_id: selectedCategoryList.map(({ id }) => id),
+      };
+
+      axios
+        .post('/user', userDataForBE)
+        .then((res) => {
+          const { accessToken, refreshToken } = res.data;
+          // 브라우저 쿠키에 access 토큰, refresh 토큰 저장
+          document.cookie = `accessToken=${accessToken}; path=/; max-age=3600; secure; SameSite=Lax;`;
+          document.cookie = `refreshToken=${refreshToken}; path=/; max-age=${7 * 24 * 60 * 60}; secure; SameSite=Lax;`;
+          navigate('/');
+        })
+        .catch((err) => console.log(err));
+    } else {
+      showModal({
+        content: `선택하지 않은 내용이 있어요. \n모든 내용을 선택해주세요.`,
+        type: 'alert',
+      });
+      return;
+    }
+  };
+
+  return (
+    <>
+      <Header>
+        <span className="title">회원가입</span>
+      </Header>
+      <NickNameContainer>
+        <InputHeader text={'닉네임'} isDetail={false} />
+        <TextFieldStyle>{'nickname'}</TextFieldStyle>
```

Comment:
일반적으로 컴포넌트의 props에 문자열을 넘겨줄때는 중괄호를 사용하지 않지만 사용하는 것도 상관은 없다고 생각하는데요.
children으로 문자열을 넘길 때 중괄호를 쓰는 건 좋지 않은 것 같습니다.
```suggestion
        <InputHeader text='닉네임' isDetail={false} />
        <TextFieldStyle>nickname</TextFieldStyle>
```

Reply:
넵! 이 부분은 [b650dcc](https://github.com/GrebbMe/grabbme-fe/pull/70/commits/b650dcc7df112572cfc7a905cd0b7612638025b6)에서 수정하였습니다! 
좀 더 꼼꼼하게 살피지 못했습니다!